def return_error():
    raise Exception
